aurora-keycloak.tf

```diff
@@ -0,0 +1,14 @@
+module "aurora_keycloak" {
+  source                    = "github.com/champ-oss/terraform-aws-aurora.git?ref=v1.0.19-099cab0"
+  cluster_identifier_prefix = var.git
+  iam_auth_lambda_enabled   = true
```

Comment:
Does this need to be enabled all the time for keycloak?

Reply:
for this module I'll say it is.  If we want to add support for other db including PostgreSQL then I'll create enhancement